index.bs

```diff
@@ -5523,6 +5558,116 @@ opened</dfn> steps given |window|, |type|, |message|, and optional |default valu
 
 </div>
 
+## The emulation Module ## {#module-emulation}
+
+The <dfn export for=modules>emulation</dfn> module contains commands and events
+relating to emulation of browser APIs.
+
+### Definition ### {#module-emulation-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+EmulationCommand = (
+  emulation.setGeolocationOverride
+)
+</pre>
+
+A <dfn>geolocation override</dfn> is a [=struct=] with an [=struct/item=] named
+<dfn attribute for="geolocation override">latitude</dfn> which is a float,
+a [=struct/item=] named <dfn attribute for="geolocation override">longitude</dfn> which is a float,
+a [=struct/item=] named <dfn attribute for="geolocation override">accuracy</dfn> which is a float,
```

Comment:
Should this be `longitudeAccuracy` or what is the difference to `altitudeAccuracy` below?

Reply:
My understanding is that [accuracy field defines accuracy to both `latitude` and `longitude`](https://w3c.github.io/geolocation/#latitude-longitude-and-accuracy-attributes). [altitudeAccuracy defines accuracy for `altitude`](https://w3c.github.io/geolocation/#altitude-and-altitudeaccuracy-attributes). That's the naming which is used in geolocation spec.

Reply:
Ah I see. So I wonder if we need a similar check if both `latitude` and `logitude` are present if the `accuracy` is set.

Reply:
`latitude` and `logitude` are mandatory fields :)